setup.py

```diff
@@ -1,11 +1,11 @@
 # -*- coding: utf-8 -*-
 
-from setuptools import setup, find_packages
+from setuptools import find_packages, setup
 
 long_desc = open('README.rst').read()
 
 setup(
-    name='dredd_hooks',
+    name='dredd-hooks-python',
```

Comment:
@w-vi @medariox This is breaking change

Reply:
@honzajavorek @w-vi 
I've sent a new PR reverting the name change, although I think you should really consider changing the PyPI name to `dredd-hooks-python`. Personally, I currently find it quite confusing to install the package with `pip install dredd_hooks` and expect it to run it as `dredd-hooks-python` from command line.

Just my opinion, at the end of the day it's your decision to make of course.

Reply:
I'm willing to think about the change, but if it happens, it needs major version bump and we need to discuss it in the [context of all the other hooks as well](https://github.com/apiaryio/dredd/search?utf8=%E2%9C%93&q=dredd_hooks&type=). And again, if the change happens, all docs need to be updated.

Would you mind to file a dedicated issue?